dotCMS/src/main/java/com/dotcms/contenttype/business/ContentTypeAPI.java

```diff
@@ -251,4 +252,27 @@
    * @throws DotDataException
    */
   Map<String, Long> getEntriesByContentTypes() throws DotDataException;
+  
+  /**
+   * Saves or update a Content Type.
```

Comment:
Please add a better javadoc explaining what if the process to follow with those fields, I think it is important to note this method can delete fields.
Example: Edit existing ones, delete the others, etc, etc